README.md

```diff
@@ -19,7 +22,7 @@ To run the clustering algorithms, the command-line tool `RunNetworkClustering` i
 The tool can be run as follows:
 
 ```
-java -cp networkanalysis.jar cwts.networkanalysis.run.RunNetworkClustering
+java -cp networkanalysis-1.1.0.jar nl.cwts.networkanalysis.run.RunNetworkClustering
```

Comment:
It is quite easy to forget about updating the version strings in this document. Can we somehow automate this?

Reply:
I have investigated this in more detail, but I have unfortunately not found a solution.

Reply:
Too bad!